duckduckgo/iOS
# Updates the way we store the secure vault encryption keys.

Task/Issue URL: https://app.asana.com/0/1203822806345703/1204823913396623/f
BSK PR: https://github.com/duckduckgo/BrowserServicesKit/pull/382

Description:
Updates the way we store the secure vault encryption keys. (See BSK PR for details)

Steps to test this PR:

Remove the app completely
- Point macOS BSK version to the `main` branch
- Create a couple of logins
- Point macOS BSK version to `daniel/keychain-fixes`
- Observe that your passwords are still available






---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
